hello/templates/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Karola's homework</title>
+</head>
+<body>
+<h1>Hi, Jerzy! This is my homework.</h1>
+<p>Here's some links you can click on:</p>
+<ul>
+  <li>
+      <a href=http://127.0.0.1:8000/hello>Here</a> you can say hello to the world
```

Comment:
Tutaj jedna uwaga, a w zasadzie to bardzi ciekawostka :) (bo jeszcze o tym nie rozmawialiśmy). Nie zahardkodowujemy endpointów w ten sposób w kodzie. Używamy tutaj znacznika języka szablonów {% url %}, ale to na spokojnie powiemy sobie o tym na jednym z kolejnych spotkań.